client/src/ui/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { InputHTMLAttributes } from 'react';
+import Close from '@material-ui/icons/Close';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  helperText?: string;
+  id: string;
+  invalid: boolean;
+  labelText: string;
+}
+
+const Checkbox = ({ id, labelText, invalid, helperText, checked, ...rest }: Props) : JSX.Element => (
+  <div className="checkbox-field">
+    <input id={id} className="checkbox-field__input" type="checkbox" {...rest} checked={checked}/>
+    <label htmlFor={id} className={`checkbox-field__label${checked ? ' checkbox-field__label--checked' : ''}${invalid ? ' checkbox-field__label--invalid' : ''} typography__label typography__label--primary`}>
```

Comment:
Why not use the `classnames` library? It should make stuff like this a bit easier and cleaner

Reply:
It might improve readability but it adds a dependency which needs to be maintained, best to keep these to a minimum especially on the client side.